examples/icestick/rs232demo/icestick.pcf

```diff
@@ -0,0 +1,11 @@
+# full iCEstick pinout:
```

Comment:
I think it would be better to have one pcf file in `icestick/` and refer to it in all examples than three copies of the (effectively) same file

Reply:
This change was initially triggered because the pcf files where using the old syntax (e.g. the  --warn-no-port). I can merge the different pcf files together at the cost of a few warnings.

Reply:
With -nowarn on any pins unused in any of the designs, there shouldn't be warnings?